src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -174,6 +174,7 @@ def upgrade_status(self) -> orchestrator.UpgradeStatusSpec:
                 r.message = 'Error: ' + self.upgrade_state.error
             elif self.upgrade_state.paused:
                 r.message = 'Upgrade paused'
+                r.is_paused = True
```

Comment:
do we also want to run `r.is_paused = True` in the above case when `self.upgrade_state.error` is true? Technically we pause the upgrade when we hit an error.

Reply:
I would say yes, it makes sense I think.

Reply:
@adk3798 I just realized that the class `UpgradeState` has `self.paused` already